README.md

```diff
@@ -2,72 +2,63 @@
 
 # Project Name
 
```

Comment:
[OPTIONAL] You can replace the `Project Name` text with your project name. Like `My Portfolio`.